File: cms/Sample/stub.cpp
#include <bits/stdc++.h>
using namespace std;

int add(int x, int y);

int main(int argc, char **argv) {
    ifstream input(argv[2]);
    ofstream output(argv[1]);

    int a, b;
    while (input >> a >> b) {
        output << add(a, b) << endl;
    }

    input.close();
    output.close();
    return 0;
}

File: cms/Sample/stub.py
import sys
from aplusb import add

input = open(sys.argv[2], 'r')
output = open(sys.argv[1], 'w')

n=int(input.readline())
for i in range(n):
    a, b = map(int, input.readline().split())
    print(add(a,b),file=output)
